pyEpiabm/pyEpiabm/tests/test_sweep/test_spatial_sweep.py

```diff
@@ -17,25 +18,29 @@ def setUpClass(cls) -> None:
         """Initialises a population with one cell and one person in
         the cell.
         """
-        cls.pop = Population()
-        cls.pop.add_cells(1)
-        cls.cell_inf = cls.pop.cells[0]
+        super(TestSpatialSweep, cls).setUpClass()
```

Comment:
Does it not work without this line? I thought it might but no worries if it doesn't (just useful for me to know)

Reply:
It works with no set up class at all (which makes sense). Initially I just removed that one line and then it failed bc I guess it was overwriting the inheritance